node/src/command.rs

```diff
@@ -402,6 +420,16 @@ pub fn run() -> Result<()> {
 				Err("Try-runtime must be enabled by `--features try-runtime`.".into())
 			}
 		},
+		// TODO: FIX ME
```

Comment:
Let's keep things simple. This pr mainly focuses on EVM support. This TODO needs some refactoring in the service module, So, I prefer to finish it in a separate pr.